bevrand.frontend/assets/js/randomizeCall.js

```diff
@@ -3,12 +3,62 @@ var randomizeList = '';
 var currentPlayList = 'tgif';
 var currentlySelectedPlayList = '';
 
+var token = "";
+var username = "";
+
 var config = {
     proxyHostname: 'https:' == document.location.protocol ? '' : 'http://localhost:4540'
 };
 
 $(document).ready(function () {
+    token = localStorage.getItem("jwt");
+    if (token) {
+        var jwtDecoded = parseJwt(token);
+        username = jwtDecoded['username'];
+        getUserPlaylist(function(userPlaylists) {
+            if (userPlaylists.length === 0) {
+                var playlistHtml = "<article>"
+                    + '<header>'
+                    + '<h3><a class="scrolly" href="/profile.html">You do not have any playlists! Create one in your Profile</a></h3>'
+                    + '</header >'
+                    + '</article >'
+
+                $('.reel').append(playlistHtml);
+            }
+            $('#navlinkLogin').hide();
+            $('#navlinkRegister').hide();
+            $('#navlinkLogout').show();
+            $('#navlinkProfile').show();
+            setGlobalVariables(userPlaylists);
+            appendPlaylistsToCarrousel(userPlaylists);
+        });
+    }
+    else {
+        getAllLists(function (playlists) {
+            $('#navlinkLogin').show();
+            $('#navlinkRegister').show();
+            $('#navlinkLogout').hide();
+            $('#navlinkProfile').hide();
+            setGlobalVariables(playlists);
+            appendPlaylistsToCarrousel(playlists);
+        });
+    }
+});
+
+$("#navlinkLogout").click(function () {
+    localStorage.setItem("jwt", "");
+
+    $('.reel').html("");
+
     getAllLists(function (playlists) {
+        $('#randomizebutton')
+            .text("Randomize!");
+        $('#randomizedDrink').hide();
+
+        $('#navlinkLogin').show();
```

Comment:
Can also be using the described toggle function above

Reply:
Not part of this PR